contracts/modules/capital/SwapOperator.sol

```diff
@@ -290,6 +294,24 @@ contract SwapOperator is ReentrancyGuard {
     emit Swapped(ETH, stETH, amountIn, amountOut);
   }
 
+  function transferToCommunityFund() external onlySwapController {
```

Comment:
New function intended for a single use. `communityFundTransferExecuted` ensures it's called only once.
Deadline's role is to prevent re-executing the transaction in case we redeploy the swap operator after a while and forget to remove the function.

Reply:
note: if we deploy within the deadline, you can still re-execute

Reply:
Yes, of course, but the chance that we'll remember is higher :)
It's intended for longer periods of time, ex redeploying in a few months (when we actually could forget haha)